scripts/enumerate_test_targets.py

```diff
@@ -0,0 +1,107 @@
+# Author: Tomas Hodan (hodantom@cmp.felk.cvut.cz)
+# Center for Machine Perception, Czech Technical University in Prague
+
```

Comment:
Please add a docstring at the beginning of this script to briefly describe what this script does ('''docstring''', such as in https://github.com/thodan/bop_toolkit/blob/master/scripts/calc_gt_info.py).